### PR TITLE
Add missing Props for Caption within Vimeo block

### PR DESCRIPTION
**Pull request by rcrphillips**
## What does this change?
Adds the required props for the caption component imported into the Vimeo block.

### CODE REVIEW

File: src/lib/content.d.ts [hunk 1]
```diff
@@ -234,9 +234,8 @@ interface VideoVimeoBlockElement {
     height: number;
     width: number;
     caption?: string;
-    credit: string;
-    title: string;
-    description: string;
```
Comment (oliverlloyd) [inline src/lib/content.d.ts:239]: We're losing `description` here, is that intentional?

File: src/web/lib/ArticleRenderer.tsx [hunk 2]
```diff
@@ -133,12 +133,14 @@ export const ArticleRenderer: React.FC<{
                     return (
                         <VimeoBlockComponent
                             pillar={pillar}
-                            url={element.url}
+                            url="https://player.vimeo.com/video/327310297?app_id=122963"
```
Comment (gtrufitt) [inline src/web/lib/ArticleRenderer.tsx:136]: Guessing you don't want every vimeo to be this one 😉

Comment (rcrphillips) [reply src/web/lib/ArticleRenderer.tsx:136]: damnit. Testing!

Comment (oliverlloyd) [inline src/web/lib/ArticleRenderer.tsx:136]: I'm guessing this slipped in by mistake?

Comment (rcrphillips) [reply src/web/lib/ArticleRenderer.tsx:136]: It is a sweet video though.....

File: src/lib/content.d.ts (hunk 1 above)
Comment (rcrphillips) [reply src/lib/content.d.ts:239]: yep

**Pull request merged**